lib/utils/emitDeprecationWarning.mjs

```diff
@@ -0,0 +1,25 @@
+import process from 'node:process';
+
+import { assertString } from './validateTypes.mjs';
+
+// Each code must be unique.
+const CODES = {
```

Comment:
You might be able to get rid of `assertString` if you set a `@type` here.

Reply:
I found `assertString` was unnecessary. Removed it via 6b4d95ee9. 👍🏼